### PR TITLE
Refactor reuseport/reuseaddr, chatbot url into separate file

### PR DESCRIPTION
**Pull request by tl676**

### CODE REVIEW

**Review by TracySun321 — APPROVED** [review]

File: basestation/base_station.py [hunk 1]
```diff
@@ -65,7 +66,7 @@ def decorated_func(*args, **kwargs):
 
 
 class BaseStation:
-    def __init__(self, app_debug=False):
+    def __init__(self, app_debug=False, reuseport = config.reuseport):
```
Comment (DoanhTran) [inline basestation/base_station.py:69]: I don't see the value of reuseport used anywhere? Don't we want to toggle the socket configuration on line 108 with this value?

Comment (tl676) [reply basestation/base_station.py:69]: thanks, fixed

**Review by DoanhTran — APPROVED** [review]
lgtm

Edit: ChatbotWrapper init doesnt init url

**Review by DoanhTran — APPROVED** [review]
lgtm fr this time

**Pull request merged**